Control_new.py

```diff
@@ -0,0 +1,160 @@
+"""
+
+"""
+import pygame
+import numpy as np
+import argparse
+import imutils
+import cv2
+
+from curve import *
+
+greenLower=(29, 86, 6)
+greenUpper=(64, 255, 255)
+ 
+class Controller(object):
+	def __init__(self):
+		self.modes=[None, 'Mouse drawing','Open CV drawing']
+		self.running_points = []
+		self.running = True
+		self.curve = None
+		self.last_space = False
+		self.last_press = False
+		self.mode = None
+
+	def handle_events(self):
+		print self.mode
+		for event in pygame.event.get():	
+			if event.type == pygame.QUIT:	# Handle window closing
+				self.running = False
+			try:
+				self.camera.release()
+				cv2.destroyAllWindows()
+			except:
+				pass
+
+		keys = pygame.key.get_pressed() # Returns a tuple of 1s and 0s corresponding to the the pressed keys
+
+		if self.mode == None:
```

Comment:
Separate this into three classes. OpenCv_Controller and Mouse_Controller can inherit from Controller. Have the video thing open in the initialization of the OpenCv class.